packages/input-container/theme/lumo/vaadin-input-container-styles.js

```diff
@@ -8,14 +8,14 @@ registerStyles(
   'vaadin-input-container',
   css`
     :host {
-      border-radius: var(--lumo-border-radius-m);
       background-color: var(--lumo-contrast-10pct);
       padding: 0 calc(0.375em + var(--lumo-border-radius-m) / 4 - 1px);
       font-weight: 500;
       line-height: 1;
       position: relative;
       cursor: text;
       box-sizing: border-box;
+      --vaadin-input-container-border-radius: var(--lumo-border-radius-m);
```

Comment:
There should be a way to set this on the `html` element instead.

```suggestion
      --vaadin-input-container-border-radius: var(--lumo-border-radius-m);
```

Reply:
Changed the theme to set `--_border-radius` instead, would that work?

Reply:
Works internally, but not as the public API. Left another comment.